data/strings/sound.txt

```diff
@@ -426,9 +499,95 @@
     tr = çıkın.
     uk = З'їзд.
     vi = Lối thoát.
-    zh-Hans = 出口.
+    zh-Hans = 驶出
     zh-Hant = 出口.
 
+  [onto]
+    comment = Used in dist_direction_onto_street in position 3 unless take_exit_number is specified. For languages that don't have an "onto" word, it must be specified as blank for that language.
+    en = onto
+    ar = إلى
+    be = на
+    cs = na
+    da = ad
+    de = auf
+    el = στην
+    es = hacia
+    es-MX = hacia
+    eu =
+    fa = ﺐﻫ
+    fi = tielle
+    fr = sur
+    hi = पर
+    hr = na
+    hu = a
+    id = ke
+    it = sulla
+    ja = 
+    ko =
+    mr = वर
+    nb = inn på
+    nl = naar
+    pl = na
+    pt = para
+    pt-BR = para
+    ro = pe
+    ru = на
+    sk = na
+    sv = in på
+    sw = kwenye
+    th = เข้าสู่
+    tr = yönünde
+    uk = на
+    vi = vào
+    zh-Hans = 上
+    zh-Hant = 上
+
+  [take_exit_number]
+    comment = Used in dist_direction_onto_street instead of "Exit" when there's an exit number after. Replaces the "onto" string as well. (There is no take_exit_number_street since this string is only used when announcing street names anyway.)
+    en = Take exit
+    ar = خروج إلى
+    be = З'езд на
+    ca = Agafa la sortida
+    cs = výstup na
+    da = Afkørsel ad
+    de = Nehmen Sie die Ausfahrt auf
+    el = Εξοδος στην
+    es = Tome la salida
+    es-MX = Tome la salida
+    eu = Irteera
+    fa = خروجﺐﻫ
+    fi = Poistu tielle
+    fr = Sortie sur
+    hi = बाहर जाएं पर
+    hr = Izlaz na
+    hu = Hajtson ki a
+    id = Keluar ke
+    it = Uscita sulla
+    ja = 出口を出る
+    ko = 출구
+    mr = बाहेर पडा वर
+    nb = Ta avkjøringen inn på
+    nl = Verlaat naar
+    pl = Wybierz zjazd
```

Comment:
@Acrobot @pm4rcin I'm happy to defer to whatever consensus exists, I just don't speak Polish so I have to judge based on your discussion. Check out https://zyphlar.github.io/organicmaps-locale-viewer/?lang=pl for some example sentences, you can click to edit and suggest changes. What seems best to both of you?

Reply:
@zyphlar from my look it all seems good right now. I have checked your webpage and everything looks good now. So I'm patiently waiting for it to be merged because it is taking too long and it would be good starting point to implement more advanced instructions as discussed throughout whole PR. :)